src/data/logos.ts

```diff
@@ -72,6 +72,7 @@ export const logos = LogoCheck({
 	gitbook: { file: 'gitbook.svg', padding: '.25em' },
 	'frontmatter-cms': { file: 'frontmatter-cms.svg', padding: '.25em' },
 	statamic: { file: 'statamic.svg', padding: '.2em' },
+	xata: { file: 'xata.svg', padding: '.8rem' },
```

Comment:
Small nit pick but the other logos have padding set in `em` so it can scale if we display the logo at different sizes. I’ve also tweaked this ever so slightly so that the logo breathes a bit more in the circle
```suggestion
	xata: { file: 'xata.svg', padding: '0.234em 0.234em 0.1875em' },
```

| Before | After |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/357379/236618060-cac6f699-a172-492c-be20-068205816c7e.png) | ![image](https://user-images.githubusercontent.com/357379/236618084-161adfae-8fdf-45c5-8afc-550b1cd039ac.png) |